Medable/ResearchStack
# PAT-1536: Removing ViewTaskActivity

### Objective
- Remove deprecated `ViewTaskActivity`
- Refactoring `ConsentViewTaskActivity` to extend `TaskActivity` instead of `ViewTaskActivity`, and renaming the class to `ConsentTaskActivity`

### How To Test
- Run the project and take any Consent Task and make sure it works until it's completed.
- Make sure the Survey tasks work as well

Credentials:
- env: int-dev
- org: abdallahandela
- study: Banadol
- Join study
- take "Consent" task and complete it.
- take any other task and complete it.

### Branches
- PAT: task/PAT-1536_remove_ViewTaskActivity
- Axon: task/PAT-1536_remove_ViewTaskActivity
- RS: task/PAT-1536_remove_ViewTaskActivity
- Cortex: development

### Links
- https://jira.devops.medable.com/browse/PAT-1536

Closes PAT-1536
